ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-ref.yaml

```diff
@@ -8,5 +8,19 @@ ref:
       memory: 600Mi
     limits:
       memory: 2Gi
+  env:
+  - name: PACKET_ACCOUNT
+    default: ""
```

Comment:
Wondering if could be useful to have a `metal-ipi` default as initial conservative approach to avoid breaking those jobs that are using directly the general workflow, like the abstract [router](https://github.com/openshift/release/blob/0017d437faeb1a00496c318c59a9f5a1b6ba1f66/ci-operator/config/openshift/router/openshift-router-master.yaml#L75-L125) one.
We are currently working to move to more specialized (per configuration) workflows, but it's not yet completed such activity.